src/lotus/tests.rs

```diff
@@ -0,0 +1,39 @@
+use url::Url;
```

Comment:
We should maybe add an issue to include a test also for the rest of the methods in this API. WDYT?

Reply:
sounds good! I will add an issue to track this.